doc/content/the-things-stack/migrate-to-v3/_index.md

```diff
@@ -108,17 +108,13 @@ To complete the migration this way, you need to have  <a href="https://github.co
 
 `ttn-lw-migrate` is used to export end devices and applications from The Things Network Stack V2 cluster to a <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/device-json/" target="_blank">JSON file</a>. This JSON file can later be <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/import-devices/" target="_blank">imported</a> in The Things Stack V3 via Console or via CLI.
 
-### Note about migrating an active session
+### Note about migrating active sessions
 
-If your The Things Stack instance version is `3.10` or higher, you may transfer the active device session as well. This means:
+Starting from The Things Stack version `v3.12.0`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 
```

Comment:
They will have to be deleted on V2

Reply:
I must have made a bad conclusion reading [this](https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/#export-end-devices-from-the-things-network-stack-v2):

> Migration from The Things Network Stack V2 to The Things Stack is one-way. LoRaWAN devices may only be handled by one Network Server at a time. The commands below will clear both the root keys and the session keys from The Things Network Stack V2 after the devices are exported. This means that the devices will no longer work in The Things Network Stack V2 .

I thought that migrating an active session means clearing those keys on V2, and that clearing those keys means you won't have to delete them on V2 because they won't work anymore anyway.